hipscat/partitioner.py

```diff
@@ -36,14 +37,18 @@ def __init__(self, catname, urls=[], fmt='csv', ra_kw='ra', dec_kw='dec',
         self.order_k = order_k
         self.verbose = verbose
         self.debug = debug
-        
+
         self.img = None
         self.orders = None
         self.opix = None
         self.hips_structure = None
         self.output_written = False
         self.threshold = None
 
+        # the order at which we will preform the margin pixel assignments
+        # needs to be equal to or greater than the highest order of partition.
+        self.highest_k = 7
```

Comment:
maybe set this as `highest_k = order_k +1` to guarantee it? In the prototype examples, the default order is set at 10 (though in practice, I don't think we see anything much higher than 7 or 8).

Reply:
Which also makes me wonder how we would specify the margin threshold in practice - in arcsec, and then convert to ~healpix order?

Reply:
1. agree
2. I was thinking about allowing the user to set it in degrees, for example `threshold = 0.1`. I tried setting it up so that the margin pixel was equal in resolution (sqrt of pixel area) to the margin threshold but that meant using a K of 512 or greater for reasonable thresholds... not really very practical 😭